docs/api/users-change-user-email.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: page
+---
+
+# Update user email
+
+The Patch request updates the user's email address but does not change the other parameters in the [`user`](user) object. 
```

Comment:
With this reference topic, would there also be others like, Update user last_name, Update user first_name?
Or, could you change the topic title to **Update user properties** and show how what you describe in this topic for the email address could apply to any of the parameters by including the parameters that you want to change in the request body?